integration-tests/msa/createMsa.test.ts

```diff
@@ -1,69 +1,70 @@
 import "@frequency-chain/api-augment";
 import assert from "assert";
 import { ApiRx } from "@polkadot/api";
-import { connect, createKeys } from "../scaffolding/apiConnection"
-import { groupEventsByKey, signPayloadSr25519 } from "../scaffolding/helpers";
+import { connect } from "../scaffolding/apiConnection"
+import { createAccount, createAndFundAccount, DevAccounts, INITIAL_FUNDING, showTotalCost, signPayloadSr25519 } from "../scaffolding/helpers";
 import { KeyringPair } from "@polkadot/keyring/types";
-import { addPublicKeyToMsa, createMsa, createSchema, deletePublicKey } from "../scaffolding/extrinsicHelpers";
+import { addPublicKeyToMsa, createMsa, createSchema, deletePublicKey, signAndSend } from "../scaffolding/extrinsicHelpers";
 import { AVRO_GRAPH_CHANGE } from "../schemas/fixtures/avroGraphChangeSchemaType";
-import { filter, firstValueFrom } from "rxjs";
 
-describe("Create Accounts", () => {
+describe("Create Accounts", function () {
+    this.timeout(150000);
+
+    const context = this.title;
+    const amount = INITIAL_FUNDING;
+    const source = DevAccounts.Alice;
+
     let api: ApiRx;
-    let keys: KeyringPair;
 
-    before(async () => {
+    before(async function () {
         let connectApi = await connect(process.env.WS_PROVIDER_URL);
         api = connectApi
     })
 
-    after(() => {
-        api.disconnect()
+    after(async function () {
+        await showTotalCost(api, context);
+        await api.disconnect()
     })
 
     // NOTE: We will need a sustainable way to create new keys for every test,
     // since there is only one node instance per test suite.
-    it("should successfully create an MSA account", async () => {
-        keys = createKeys("//Alice")
-        const chainEvents = await createMsa(api, keys)
+    it("should successfully create an MSA account", async function () {
+        const { newAccount: keys } = await createAndFundAccount({ api, amount, source, context });
+        const chainEvents = await createMsa(api, keys);
 
-        assert.equal(chainEvents["system.ExtrinsicFailed"], undefined);
         assert.notEqual(chainEvents["system.ExtrinsicSuccess"], undefined);
         assert.notEqual(chainEvents["msa.MsaCreated"], undefined);
         assert.notEqual(chainEvents["transactionPayment.TransactionFeePaid"], undefined);
-    }).timeout(15000)
+    });
 
-    it("should successfully mimic a user's path using tokens", async () => {
-        keys = createKeys("//Charlie")
+    it("should successfully mimic a user's path using tokens", async function () {
+        const { newAccount: keys } = await createAndFundAccount({ api, amount, source, context });
```

Comment:
Is there a way to simplify this to what a developer would most often use?  i.e.  just the `amount`. Also, it may be better to give the accounts a real name (like _henryAccount_ or _johnAccount_) to make it easier to read.

Reply:
Not clear on what you mean here... give the accounts a name how? Or you just mean the variable name?
And for "developer would most often use"? You mean for the arguments to `createAndFundAccount`? I suppose I could create an object containing the standard options that could simply be passed in, is that what you're after?

Reply:
The variable names.

Reply:
Reduce the number of inputs to that function.

Reply:
Can't really eliminate any of the inputs to that function, but I defined an interface to encapsulate it & set up a default instance of it at test suite initialization to simplify things. See what you think.